deploy/manager/manager.yaml

```diff
@@ -3,15 +3,27 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: governance-policy-propagator
+  labels: 
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: webhook
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/part-of: webhook
+    control-plane: controller-manager
```

Comment:
Do we need to add any of these labels for the webhook to work? If so, let's make them a bit more specific to just our controller, if possible. Eg `app.kubernetes.io/name: governance-policy-propagator`

Reply:
let me try! last time i removed then didn't work

Reply:
app.kubernetes.io/name: looks like they use this as a resource kind

Reply:
seems like woking well without label